cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1484,7 +1484,8 @@ def test_shared_storage_name_validator(name, expected_message):
         ("shared_ebs_1", None),
         ("shared", None),
         ("/shared", None),
-        ("home", "mount directory .* is reserved"),
+        ("home", None),
```

Comment:
Shouldn't this be `/home` rather than `home`?

Reply:
no it's not necessary, the validator adds the slash 
```
if not mount_dir.startswith("/"):
            mount_dir = "/" + mount_dir
```